oauthenticator/azuread.py

```diff
@@ -6,23 +6,16 @@
 import json
 import jwt
 import os
-import re
-import string
 import urllib
-import sys
 
 from tornado.auth import OAuth2Mixin
 from tornado.log import app_log
-from tornado import web
-
-from tornado.httputil import url_concat
 from tornado.httpclient import HTTPRequest, AsyncHTTPClient
 
 from jupyterhub.auth import LocalAuthenticator
 
-from traitlets import List, Set, Unicode
+from traitlets import Unicode, default
 
-from .common import next_page_from_links
```

Comment:
Just to double check: all these removed imports were unused ones or is there something to double check here?

Reply:
yeah they were all unused - at least according to PyCharm! Thanks